readme.md

```diff
@@ -21,19 +20,22 @@ For IoT Hub Management SDK in .NET, please visit [azure-sdk-for-net](https://git
 Visit [Azure IoT Dev Center][iot-dev-center] to learn more about developing applications for Azure IoT.
 
 ## Samples
-Samples are available at [Azure IoT Samples for C#](https://github.com/Azure-Samples/azure-iot-samples-csharp)
+In the repository, you will find a set of samples that will help you get started:
+* [IoT Hub Device SDK samples](./iothub/device/samples)
```

Comment:
Why is this changed

Reply:
You have to revert this. this is incorrect

Reply:
I think you picked up readme from your private repo and changes in latest is what you need. Can you please revert readme to what is reflected in master?

Reply:
Fixed.